packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -234,6 +237,13 @@ export class EntityService {
         `The entity name and plural display name cannot be the same.`
       );
     }
+    if (isReservedName(args.data?.name?.toLowerCase().trim())) {
+      throw new AmplicationError(
+        `"${args.data?.name
+          ?.toLowerCase()
+          .trim()}" is a reserved name and cannot be used.`
```

Comment:
Create a new Error and use a string template in the contractor to generate the message
see this example 
packages/amplication-server/src/core/app/ReservedEntityNameError.ts